integrations/lambdatest_integration_demo_project/lambdatest_integration_demo_project/android/app/build.gradle

```diff
@@ -146,11 +146,18 @@ android {
         }
     }
     signingConfigs {
-        debug {
-            storeFile file('debug.keystore')
-            storePassword 'android'
-            keyAlias 'androiddebugkey'
-            keyPassword 'android'
+        release {
+            if (System.getenv()["CI"]) { // CI=true is exported by Codemagic
+                storeFile file(System.getenv()["CM_KEYSTORE_PATH"])
+                storePassword System.getenv()["CM_KEYSTORE_PASSWORD"]
+                keyAlias System.getenv()["CM_KEY_ALIAS_USERNAME"]
+                keyPassword System.getenv()["CM_KEY_ALIAS_PASSWORD"]
+            } else {
+                storeFile file("/path/to/local/myreleasekey.keystore")
+                storePassword "password"
+                keyAlias "MyReleaseKey"
+                keyPassword "password"
+            }
```

Comment:
I don't think we should encourage people to expose passwords that would be committed to source control. Maybe use a similar syntax to show reading the values from the local machine environment instead.

Reply:
Well, I do not think that else condition is used anyways. We can get rid of it as well.

Reply:
What if somebody wants to run a build on their local machine?

Reply:
I added a comment on this inside the yaml file. If they want to configure their code signing differently, then key.properties way of doing it is possible, I guess. I added a link to the docs as well.

Reply:
Added the changes.